products/ubuntu2004/profiles/stig.profile

```diff
@@ -458,6 +458,8 @@ selections:
     # UBTU-20-010413 The Ubuntu operating system must disable kernel core dumps  so that it can fail to a secure state if system initialization fails, shutdown fails or aborts fail.
     - service_kdump_disabled
 
+    # UBTU-20-010444 Ubuntu operating system must implement cryptographic mechanisms to prevent unauthorized modification of all information at rest.
```

Comment:
please revert this commit or better yet drop it while rebasing.

Reply:
Dropped!